### PR TITLE
Style filter button and DnD support, fix empty value

### PR DESCRIPTION
**Pull request by ivanmjartan**
JIRA: RAIL-4422

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: libs/sdk-ui-filters/src/AttributeFilter@next/Components/DropdownButton/AttributeFilterDropdownButton.tsx [hunk 1]
```diff
@@ -70,10 +85,12 @@ export const AttributeFilterDropdownButton: React.VFC<IAttributeFilterDropdownBu
                     "gd-is-filtering": isFiltering,
                     "gd-is-active": isOpen,
                     "gd-is-loaded": isLoaded,
+                    "gd-is-draggable": isDraggable,
                 },
             )}
             onClick={onClick}
         >
+            {icon && <div className="gd-attribute-filter-dropdown-button-icon__next"> {icon}</div>}
```
Comment (no23reason) [inline libs/sdk-ui-filters/src/AttributeFilter@next/Components/DropdownButton/AttributeFilterDropdownButton.tsx:93]: Is the space before the `{icon}` intentional?

Comment (ivanmjartan) [reply libs/sdk-ui-filters/src/AttributeFilter@next/Components/DropdownButton/AttributeFilterDropdownButton.tsx:93]: fixed

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2897

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2673/ : FAILURE in 10m 10s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2974/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1809/ : SUCCESS in 17m 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11333/ : SUCCESS in 19m 54s

**Pull request merged**